package.json

```diff
@@ -38,10 +38,10 @@
   "private": true,
   "scripts": {
     "clean": "yarn lerna run clean --parallel",
-    "build": "yarn lerna run build",
-    "test": "yarn lerna run test --parallel",
+    "build": "npx nx run-many --target=build --all --parallel 3",
+    "test": "npx nx run-many --target=build --all --parallel 3",
+    "lint": "npx nx run-many --target=lint --all --parallel 3",
     "test:coverage": "yarn lerna run test:coverage --parallel",
-    "lint": "yarn lerna run lint",
     "lint:ts:check": "yarn lerna run lint:ts:check",
```

Comment:
It looks like there is still some lerna stuff in here, we would also want to replace that as well?

Reply:
Yea nx and tsup can replace lerna and that would speed up even cold builds from scratch

Reply:
Would do this in a seperate pr since lerna does some other stuff like publish npm packages that would need to be replaced too

Reply:
actually we aren't using lerna for that I was mistaken so nuked lerna now